query-engine/driver-adapters/pnpm-workspace.yaml

```
@@ -0,0 +1,8 @@
+packages:
+  - '../../../prisma/packages/adapter-libsql'
+  - '../../../prisma/packages/adapter-neon'
+  - '../../../prisma/packages/adapter-pg'
+  - '../../../prisma/packages/adapter-planetscale'
+  - '../../../prisma/packages/driver-adapter-utils'
+  - '../../../prisma/packages/debug'
```

So you don't need to add it in the future

```suggestion
  - '../../../prisma/packages/*'
```

---

Wouldn't this require to build all the packages in the prisma monorepo each time I rebuild driver adapters?

---

I would assume that pnpm would only install the deps that you are actually using, the best is to give it a try. Looks good either way.